projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -2759,3 +2771,90 @@ function wpcom_launchpad_get_latest_draft_id() {
 
 	return $cached_draft_id;
 }
+
+/**
+ * Check if the SSL task should be disabled.
+ *
+ * @return bool Will return true if the primary domain is a WPCOM domain.
+ */
+function wpcom_launchpad_is_ssl_task_disabled() {
```

Comment:
I'll need to use the same logic to check to see if the site has a primary domain that is *not* a WPCOM domain. :D Would it make sense to make this a generic function like `wpcom_launchpad_is_primary_domain_wpcom()`? Or move the domain-checking logic to a generic helper function?

Reply:
yes, I think it makes sense to do a function specifically for this. 
I'll look around a bit to see where this function might be a better fit.

Reply:
I ended up adding the `wpcom_launchpad_is_primary_domain_wpcom` function. I tried to find a better place, but couldn't :)